contracts/hardhat.config.js

```diff
@@ -42,8 +42,9 @@ const {
 
 const MAINNET_DEPLOYER = "0x71F78361537A6f7B6818e7A760c8bC0146D93f50";
 // Mainnet contracts are governed by the Governor contract (which derives off Timelock).
-const MAINNET_GOVERNOR = "0x8e7bDFeCd1164C46ad51b58e49A611F954D23377";
-const MAINNET_MULTISIG = "0xe011fa2a6df98c69383457d87a056ed0103aa352";
+const MAINNET_GOVERNOR = "0x830622BDd79CC677eE6594E20bBda5B26568b781";
+// Multi-sig that controls the Governor. Aka "Guardian".
+const MAINNET_MULTISIG = "0xbe2AB3d3d8F6a32b96414ebbd865dBD276d3d899";
```

Comment:
I can't double check this one but someone should.

Reply:
Good to double check. This can be checked on-chain (by running npx hardhat debug or on etherscan) by looking at the governor address set on all the ousd contracts (it should be set to 0x830622BDd79CC677eE6594E20bBda5B26568b781). Then check the governor's contract admin address is set to the new multisig 0xbe2AB3d3d8F6a32b96414ebbd865dBD276d3d899.

Reply:
This comment wasn't on governor, but the multisig.  And by checking it, I would say checking that it's a multisig we control via information I'm not privy to.  If you're happy with it, I'm happy with it.   I just can't verify it .

Reply:
Thanks. Yes, I double checked in our internal doc as well as on the ousd registry at https://docs.ousd.com/smart-contracts/registry that 0xbe2AB3d3d8F6a32b96414ebbd865dBD276d3d899 is our 5/8 multi-sig.